### PR TITLE
fixed next button focus

### PR DESCRIPTION
**Pull request by siraman**
When you enter the URL, the enter key press will make user name and password visible.
Once you enter the password, enter key press will send the request.
Enter key press on username field will NOT send the request.

### CODE REVIEW

**Review by LinusCastelino — COMMENTED** [review]
Let's discuss the approach tmrw or anytime we meet next. Need to understand the idea behind the changes introduced

File: src/main/react-front-end/src/views/Transfer/EndpointAuthenticateComponent.js [hunk 1]
```diff
@@ -264,17 +265,25 @@ export default class EndpointAuthenticateComponent extends Component {
...
-    		});
-    	}
+	const {url, username, password, needPassword} = this.state;
+	if(url.substr(url.length - 3) === '://') {
+		this._handleError("Enter the correct URL")
+		return
+	}
+	if(!needPassword){
+		this.endpointCheckin(this.state.url, this.state.portNum, {}, () => {
+			console.log('setting need pwd to ture..')
```
Comment (LinusCastelino) [inline src/main/react-front-end/src/views/Transfer/EndpointAuthenticateComponent.js:341]: This console log can be removed

**Comment by siraman** [comment]
Update:
1. If the URL is empty, error message is shown as "Enter correct URL".
2. On Enter press will do the validation 
3. Username and password validation done on Enter key and also on Next button press.

**Review by LinusCastelino — COMMENTED** [review]
Code changes look good as per requirement - error handling done well.
Will approve and merge after a demo in meeting.

Suggestion - On key press = 'Enter' can we trigger the click event of the 'Next' button instead of calling the authFunction()? This will give the button click feel on UI (which I guess is not the case with the current way it has been coded)

**Comment by siraman** [comment]
Update: Incorporated the suggested changes.

**Comment by LinusCastelino** [comment]
@siraman @KiranPrabhakar135 
any update on this?

**Pull request merged**